proto/src/rangeserver.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package rangeserver;
+
+message RangeId {
+    string keyspace_id = 1;
+    string range_id = 2;
+}
+
+service RangeServer {
+    // Our PreFetch rpc accepts a PreFetchRequest and returns HelloReplies
+    rpc PreFetch (PreFetchRequest) returns (HelloReply);
+}
+
+message PreFetchRequest {
```

Comment:
nit: I prefer Prefetch as one word, as opposed to PreFetch

Reply:
Updated